name: Test continue on error
on:
  push:
  workflow_dispatch:
jobs:
  continue:
    runs-on: ubuntu-latest
    steps:
      - name: exit 1
        continue-on-error: true
        run: exit 1
      - run: exit 0
      - name: the second exit 1
        continue-on-error: true
        run: exit 1
  continue2:
    runs-on: ubuntu-latest
    steps:
      - name: the second exit 1
        continue-on-error: true
        run: exit 1
      - run: exit 0
  # fail:
  #   runs-on: ubuntu-latest
  #   continue-on-error: false
  #   steps:
  #     - run: exit 1
  # fail2:
  #   runs-on: ubuntu-latest
  #   continue-on-error: false
  #   steps:
  #     - run: exit 0
  # fast:
  #   runs-on: ubuntu-latest
  #   strategy:
  #     fail-fast: false
  #   steps:
  #     - run: exit 1
  # fast2:
  #   runs-on: ubuntu-latest
  #   strategy:
  #     fail-fast: false
  #   steps:
  #     - run: exit 0
  # fft:
  #   runs-on: ubuntu-latest
  #   strategy:
  #     fail-fast: true
  #   steps:
  #     - run: exit 1      
  # fft2:
  #   runs-on: ubuntu-latest
  #   strategy:
  #     fail-fast: true
    # steps:
    #   - run: exit 0
        
      
